frontend/src/pages/article/components/article-view/article-view.tsx

```diff
@@ -107,12 +106,16 @@ const ArticleView: React.FC<Properties> = ({
               iconClassName={styles.icon}
               onClick={onButtonClick}
             />
-            <IconButton
-              iconName="comment"
-              className={styles.iconButton}
-              iconClassName={styles.icon}
-              onClick={onButtonClick}
-            />
+            <Link
+              to={`#${LinkHash.COMMENTS}` as typeof AppRoute.ARTICLES_$ID}
```

Comment:
Let's try it this way
```suggestion
              to={{ hash: LinkHash.COMMENTS }}
```